src/prefect/cli/global_concurrency_limit.py

```diff
@@ -60,15 +60,17 @@ async def list_global_concurrency_limits():
     table.add_column("Updated", style="blue", no_wrap=True)
 
     for gcl_limit in sorted(gcl_limits, key=lambda x: f"{x.name}"):
+        assert gcl_limit.created is not None, "created is not None"
+        assert gcl_limit.updated is not None, "updated is not None"
         table.add_row(
             str(gcl_limit.id),
             gcl_limit.name,
             str(gcl_limit.active),
             str(gcl_limit.limit),
             str(gcl_limit.active_slots),
             str(gcl_limit.slot_decay_per_second),
-            pendulum.instance(gcl_limit.created).isoformat(),
-            pendulum.instance(gcl_limit.updated).diff_for_humans(),
+            pendulum_instance(gcl_limit.created).isoformat(),
+            pendulum_instance(gcl_limit.updated).diff_for_humans(),
```

Comment:
In https://github.com/PrefectHQ/prefect/pull/17074 I created a `human_friendly_diff` method for this. I think this will work best if we don't rely on any methods on `pendulum.DateTime` because it'll make switching easier.